### DISCUSSION TITLE
My VS Code does not detect @chakra-ui/react library

### DISCUSSION THREAD

**Discussion by iyansr** [chakra-ui/chakra-ui#3656] [Q&A] [answered]
My VS Code does not detect @chakra-ui/react library , but its running well when I start dev

![image](https://user-images.githubusercontent.com/42711013/112259138-54c54a00-8c9a-11eb-8a94-b08c70541cf7.png)

package.json : 

```json
{
  "name": "web-appointment",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@chakra-ui/react": "^1.4.1",
    "@chakra-ui/theme": "^1.7.1",
    "@chakra-ui/theme-tools": "^1.1.1",
    "@emotion/react": "11",
    "@emotion/styled": "11",
    "@testing-library/jest-dom": "^5.11.4",
    "@testing-library/react": "^11.1.0",
    "@testing-library/user-event": "^12.1.10",
    "framer-motion": "4",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-scripts": "4.0.3",
    "web-vitals": "^1.0.1"
  },
  "scripts": {
    "start": "react-app-rewired start",
    "build": "react-app-rewired build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "react-app-rewire-alias": "^1.0.2",
    "react-app-rewired": "^2.1.8"
  }
}

```

**Comment by with-heart** [comment]
Hi @iyansr! I've created a CRA project using the same `package.json` file as you, and it's working as expected:

![image](https://user-images.githubusercontent.com/1954752/112304235-a1e1f580-8c73-11eb-88e9-b2730c8ba28e.png)

Is there any way you could share your repo with us so we can investigate what else might be causing that?

**Reply by iyansr** [reply]
hi @with-heart , thanks for answer!
Here is my repo : https://github.com/iyansr/sample

**Comment by iyansr** [comment, marked as answer]
Update : 
- I've managed to make it work by removing `"target": ""es6"` in `jsconfig.json`

Before: 
![image](https://user-images.githubusercontent.com/42711013/112308237-81965300-8cd4-11eb-937a-8098154a74d9.png)

After:
![image](https://user-images.githubusercontent.com/42711013/112308265-89ee8e00-8cd4-11eb-95ce-b16f615a82d7.png)

Thanks @with-heart for answering my question.

I dont know if this VSCode Problem or anything.

But happy to know my problem 😅😁
